packages/Panel/src/components/Header/Header.styles.js

```diff
@@ -2,32 +2,35 @@ import styled, { css } from "styled-components";
 import { spacer } from "@paprika/stylers/lib/helpers";
 import stylers from "@paprika/stylers";
 import tokens from "@paprika/tokens/lib/tokens";
-import Button from "@paprika/button";
+import * as types from "../../types";
 
-const typeKinds = {
-  [Button.types.kind.DEFAULT]: `background: ${tokens.color.white}; color: ${tokens.color.black};`,
-  [Button.types.kind.PRIMARY]: `background: ${tokens.diligent.actionPrimary}; color: ${tokens.color.white};`,
+const sizeStyles = {
+  [types.sizes.MEDIUM]: css`
+    height: 18px;
+    padding: ${tokens.spaceLg} ${stylers.spacer(2)};
+
+    [data-pka-anchor="heading"] {
+      margin: 0;
+    }
+  `,
+  [types.sizes.LARGE]: css`
+    height: 23px;
+    padding: ${stylers.spacer(2)} ${stylers.spacer(3)};
+
+    [data-pka-anchor="heading"] {
+      margin: 0;
+    }
```

Comment:
I think we can pull these out as it's been applied in L45-49. Same thing applies to L12-14.